sympy/physics/vector/dyadic.py

```diff
@@ -539,6 +539,51 @@ def _eval_evalf(self, prec):
             new_args.append(tuple(new_inlist))
         return Dyadic(new_args)
 
+    def xreplace(self, rule):
+        """
+        Replace occurrences of objects within the measure numbers of the Dyadic.
+
+        Parameters
+        ==========
+
+        rule : dict-like
+            Expresses a replacement rule.
+
+        Returns
+        =======
+
+        Dyadic : Result of the replacement.
```

Comment:
This still doesn't quite follow numpydoc conventions: https://numpydoc.readthedocs.io/en/latest/format.html#sections